resources/lib/person.py

```diff
@@ -64,11 +66,26 @@ def get_person_details(self):
 
         return li
 
+    def get_combined_list(self, movies, tvshows):
+        combined = movies + tvshows
+        combined = sort_dict(combined,'release_date',True)
```

Comment:
TV shows have no `release_date` key. It's `first_air_date`. Sorting like this will not work. You have to merge both keys to a temporary one before that can be used for sorting.

Reply:
Actually you look at the code line 155 I copy first_air_date into a new key release_date so that sorting works perfectly.

Reply:
Ah, thanks. Overlooked it.